File: Punto 1/clasificados40/src/ttps/clasificados/Usuario.java
package ttps.clasificados;

public class Usuario {
	private String username, password, tipo;
	
	public Usuario (String username, String password, String tipo) {
		this.setUsername(username);
		this.setPassword(password);
		this.setTipo(tipo);
	}
	
	public void setUsername(String username) {
		this.username = username;
	}
	
	public void setPassword(String password) {
		this.password = password;
	}
	
	public String getUsername() {
		return this.username;
	}
	
	public String getPassword() {
		return this.password;
	}

	public String getTipo() {
		return tipo;
	}

	public void setTipo(String tipo) {
		this.tipo = tipo;
	}
	
}

File: FoodTrucks/src/app/modelos/usuario/usuario.ts
export class Usuario {
    public nombre: string;
    public apellido: string;
    public username: string;
    public password: string;
    //public tipocuenta: string;

    constructor (nombre: string, apellido: string, username: string, password: string){
        this.nombre = nombre;
        this.apellido = apellido;
        this.username = username;
        this.password = password;
    }
    
}
